### PR TITLE
pem: 8.5 release

### PR DESCRIPTION
**Pull request by ccestes**
Added "Autocreated" rows and "Yes" for the corresponding alerts per [PEM-4436](https://enterprisedb.atlassian.net/browse/PEM-4436).

I did not add "Autocreated" rows for every table because most of them did not have any such alerts, but I can add them if they are needed to be consistent. 

This branch is from v8 but will be added to v8.5.

### CODE REVIEW

File: product_docs/docs/pem/8/considerations/authentication_options/configuring_the_pem_server_to_use_windows_kerberos_server.mdx [hunk 1]
```diff
@@ -170,19 +170,19 @@ It will display the principal along with the kerberos ticket.
 
 -   Run the PEM configure script on the PEM Server to use Kerberos Authentication
 
-    ```text
+    ```ini
```
Comment (drothery-edb) [inline product_docs/docs/pem/8/considerations/authentication_options/configuring_the_pem_server_to_use_windows_kerberos_server.mdx:173]: @ccestes This one is a script/command that you run in the terminal. It should be shell not ini.

**Comment by nidhibhammar** [comment]
@ccestes, I have added some content in pem_alert_templates file and also added the content from PR 2747 to this PR as everything will go out together with PEM 8.5 release. You can refer to the changes done in the last commit and close the PR 2747 with your comment if everything looks good.

**Comment by drothery-edb** [comment]
@nidhibhammar , is there a reason why 2747 can't be merged now? It doesn't seem specific to 8.5, but I might be missing something. Doesn't the new note added in 2747 describe existing behavior?

**Comment by nidhibhammar** [comment]
@drothery-edb, I have done the same changes in PEM 8.5 Online (rst) files as well to keep the content consistent, hence thought of publishing the 2747 changes along with the 8.5 release which is going out this week only.

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**